pkg/login/social/connectors/signature.go

```diff
@@ -0,0 +1,30 @@
+package connectors
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+
+	"github.com/go-jose/go-jose/v3"
+)
+
+func SignToken(data []byte, privateKey *ecdsa.PrivateKey) (string, error) {
```

Comment:
I don't see this function being used in this PR. Should this be part of a separate PR?

Reply:
Yep, it's been added accidentally.

Reply:
Removed